crates/wit/src/parsed_type/foreign_mod_prolog.rs

```diff
@@ -76,11 +77,11 @@ impl ForeignModPrologGlueCodeGenerator for Vec<(String, ParsedType)> {
                 arg_names.push(arg_ident.clone());
 
                 match ty {
-                    ParsedType::Utf8String => {
+                    ParsedType::Utf8String(PassingStyle::ByValue) => {
```

Comment:
maybe add a comment on the meaning of the matched variants? and on `_ => {}` would be nice too!